app/helpers/application_helper.rb

```diff
@@ -24,6 +24,42 @@ def miq_accordion_panel(title, condition, id, &block)
     end
   end
 
+  def single_relationship_link(record, table_name, property_name = nil)
+    out = ''
+    property_name ||= table_name
+    ent = record.send(property_name)
+    name = ui_lookup(:table => table_name.to_s)
+    if role_allows(:feature => "#{table_name}_show") && !ent.nil?
+      out = content_tag(:li) do
+        link_to("#{name}: #{ent.name}",
+                {:controller => table_name, :action => 'show', :id => ent.id.to_s},
+                :title => _("Show this #{record.class.name.demodulize.titleize}'s parent #{name}"))
+      end
+    end
+    out
+  end
+
+  def multiple_relationship_link(record, table_name)
+    out = ''
+    if role_allows(:feature => "#{table_name}_show_list") &&
+       (table_name != 'container_route' || record.respond_to?(:container_routes))
+      plural = ui_lookup(:tables => table_name.to_s)
+      count = record.number_of(table_name.to_s.pluralize)
+      if count == 0
+        out = content_tag(:li, :class => "disabled") do
+          link_to("#{plural} (0)", "#")
+        end
+      else
+        out = content_tag(:li) do
+          link_to("#{plural} (#{count})",
+                  {:action  => 'show', :id => @record, :display => table_name.to_s.pluralize},
+                  :title => _("Show #{plural}"))
+        end
+      end
+    end
+    out
+  end
+
```

Comment:
I'm guessing these functions should be in somewhere like ```listnav_helper.rb``` the same way ```textual_link``` is in ```textual_summary_helper.rb```
@matthewd ?